charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -149,13 +155,18 @@ spec:
         args:
           - /bin/sh
           - -c
-          - wget https://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/1.6.2/jolokia-jvm-1.6.2-agent.jar && mv jolokia-jvm-1.6.2-agent.jar /config
+          - wget -o /tmp/jolokia-jvm-1.6.2-agent.jar https://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/1.6.2/jolokia-jvm-1.6.2-agent.jar; mv /tmp/jolokia-jvm-1.6.2-agent.jar /config/jolokia-jvm-1.6.2-agent.jar
```

Comment:
could we get away with just doing this:
```
- wget -o /config/jolokia-jvm-1.6.2-agent.jar https://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/1.6.2/jolokia-jvm-1.6.2-agent.jar
```
If we can download the jarfile into /tmp, and then move it into /config, I'm not seeing why we can't just download it staright to /config in the first place.

Reply:
I don't see why not, but if we do keep them separate, I'd like to see the `mv` on its own line. I almost didn't see it way over there.

Reply:
I tested this change and push the commit. 

It also worth noting that when we have management-api support in Medusa the `get-jolokia` initContainer will be removed.